### ISSUE TITLE
AdminJS is not defined in .entry.js

### ISSUE THREAD

**Issue by malinowskil123** [SoftwareBrothers/adminjs#871]
When running adminjs with a custom component the auto-generated `.entry.js` errors out and doesn't render the custom react component.
 
* adminjs@5.0.0
* @adminjs/express@4.0.0
* @adminjs/typeorm@2.2.0

```
.entry.js:3 Uncaught ReferenceError: adminjs is not defined
    at .entry.js:3 
```

I also get this warning in the console from my backend app:
P.S. I've noticed you guys use `@rollup/plugin-replace` in this bundler file `adminjs/src/backend/bundler/config.js`
```
@rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recommended to set this option to `true`, as the next major version will default this option to `true`.
No name was provided for external module 'adminjs' in output.globals – guessing 'adminjs'
```

This is how I am loading the component in the config object 
```
dashboard: {
      handler: async () => {
        return { some: 'output' }
      },
      component: AdminJS.bundle('./my-dashboard-component'),
    },
```

**Comment by anuragmath** [comment]
Yes, I m also having similar trouble while trying to import and use adminjs in a custom component.

**Comment by dziraf** [comment]
This is fixed in `5.0.1`
